File: JAVASCRIPT PROGRAMMING/Test59.js
"use strict";
// 59. PRINT THE DIGITS OF A NUMBER (INPUT: 153 OUTPUT: 3 5 1)
const digitOfNumber = (input) => {
    while (input !== 0) {
        let rem = input % 10;
        process.stdout.write(`${rem} `);
        input = Math.floor(input / 10);
    }
    console.log();
};
digitOfNumber(153);
digitOfNumber(1234);

File: PYTHON PROGRAMMING/Test59.py
# 59. REVERSE NUMBER
from math import *
def Reverse(num):
    rev = 0
    NUM = num
    while num != 0:
        rem = num % 10
        rev = (rev * 10) + rem
        num = floor(num / 10)
    print(f"Reverse Of {NUM} Is: {rev}")
Reverse(123)
Reverse(1234)
Reverse(12345)